### PR TITLE
Wait for WebSocket before proceeding with tests

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: app/test_util.py [hunk 1]
```diff
@@ -38,6 +37,12 @@ def server_url(self, value):
 
     def get(self, uri):
         return webdriver.Firefox.get(self, self.server_url + uri)
+        webdriver.Firefox.get(self, self.server_url + uri)
```
Comment (acrellin) [inline app/test_util.py:40]: How can this ever be executed if it's right after a return statement?

Comment (stefanv) [reply app/test_util.py:40]: Yeah, that return is supposed to go.  Didn't grab that line of the patch, will do.

**Comment by stefanv** [comment]
@ari Fixed the merge conflict.

**Comment by ari** [comment]
Who are you?

**Comment by stefanv** [comment]
We meet the real @ari! :) 

Sorry for that, @ari, I was trying to tag teammate Ari who is @acrellin.

**Pull request merged**